### PR TITLE
Update the user guide in accordance with the new feature specification

### LINKED ISSUE #64: Update UG with newly discussed minimum required features
Update the UG to match newly discussed minimum required features. I understand that it would be a pain to keep changing as we develop the features, and I don't intend for us to do that. Just add the minimum required features this time, then we all have something to refer to for the foreseeable future.

### PR DESCRIPTION
**Pull request by aureliony**
Closes #64

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -54,20 +54,17 @@ AssetBook-3 is a desktop application for logistics managers to keep track point-
...
+Adds a new contact to the system, with 0 or more assets associated with the contact.
 
-Format for adding a new contact: `add --name <name> [--email <email> --phone <phone>] [--tag <tag>]...`
-
-Format for adding a new asset: `add --id <asset id> --location <location>`
+Format: `add --name <name> [--email <email>] [--phone <phone>] [--address <address>] [--asset <asset>]...`
 
 <box type="tip" seamless>
 
 **Tip:** A person can have any number of tags (including 0)
```
Comment (yisiox) [inline docs/UserGuide.md:63]: Should we remove mention of tags?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -86,57 +83,64 @@ Format for adding a new asset: `add --id <asset id> --location <location>`
...
-* Not necessarily unique.
+`--address`
+* Address of the contact.
 
-`--location`
-* The location associated with the asset.
+`--asset`
+* Asset(s) associated with contact.
+* Contact can be created first without assets, then assets can be added later using the Edit command.
+* Case sensitive, i.e. NUS != nus.
```
Comment (yisiox) [inline docs/UserGuide.md:92]: Nit: should we use computing notation `!=` in a user guide?

**Review by yisiox — COMMENTED** [review]
LGTM overall, but perhaps some nits to fix

File: docs/UserGuide.md (hunk 1 above)
Comment (aureliony) [reply docs/UserGuide.md]: Replaced with assets

File: docs/UserGuide.md (hunk 2 above)
Comment (aureliony) [reply docs/UserGuide.md]: Replaced with ≠

File: docs/UserGuide.md [hunk 3]
```diff
@@ -86,57 +83,64 @@ Format for adding a new asset: `add --id <asset id> --location <location>`
...
 
 ### Delete a contact or asset : `delete`
 
-Delete a contact or asset from the system by specifying its ID.
+Delete a contact from the system by specifying its ID.
 
 Format: `delete <id>`
-* The id refers to the unique id shown in the GUI.
+* The id refers to the unique contact id shown in the GUI.
+* The asset(s) associated with the contact will not be deleted.
```
Comment (yisiox) [inline docs/UserGuide.md:104]: Should this be the case?

**Review by yisiox — COMMENTED** [review]
I think we shouldn't use the term `id` to refer to contacts but rather it works by index.

File: docs/UserGuide.md (hunk 3 above)
Comment (aureliony) [reply docs/UserGuide.md:104]: Yes, the asset would remain in the `HashMap`, so we don't need to track the number of people using each asset.

**Review by yisiox — APPROVED** [review]
LGTM

**Pull request merged**